Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.py

```diff
@@ -280,25 +278,26 @@ def main() -> None:  # pragma: no cover
     verify_certificate = not params.get('insecure', False)
     proxy = params.get('proxy', False)
     first_fetch = params.get('first_fetch')
-    max_fetch = min(arg_to_number(params.get('max_fetch')), MAX_EVENTS_PAGES_PER_FETCH)  # type: ignore[type-var]
+    max_fetch = arg_to_number(params.get('max_fetch', 1000))
     vendor, product = params.get('vendor', 'netskope'), params.get('product', 'netskope')
 
     demisto.debug(f'Command being called is {demisto.command()}')
     try:
         client = Client(base_url, token, api_version, verify_certificate, proxy)
 
         last_run = demisto.getLastRun()
-        first_fetch = int(arg_to_datetime(first_fetch).timestamp())  # type: ignore[union-attr]
-        last_run = {}
         for event_type in ALL_SUPPORTED_EVENT_TYPES:
-            last_run_id_key = f'{event_type}-ids'
-            last_run[event_type] = last_run.get(event_type, first_fetch)
-            last_run[last_run_id_key] = last_run.get(last_run_id_key, [])
-        demisto.debug(f'Initialize last run: {last_run}')
+            # First Fetch
+            if not last_run.get(event_type):
+                first_fetch = int(arg_to_datetime(first_fetch).timestamp())  # type: ignore[union-attr]
+                last_run_id_key = f'{event_type}-ids'
+                last_run[event_type] = last_run.get(event_type, first_fetch)
```

Comment:
Due to your initial `if` statement thie: `last_run.get(event_type)` will always return None, so why not do just this and that's it? `last_run[event_type] = first_fetch`

Reply:
Not sure I understand this comment. Let's sync on it today as well